### ISSUE TITLE
Bug: 自动重试时 negative_prompt 丢失

### ISSUE THREAD

**Issue by MirrorCY** [koishijs/novelai-bot#225]
```
2024-02-04 19:46:44 [I] loader reload plugin novelai:eatokh
2024-02-04 19:46:46 [I] novelai request { model: 'nai-diffusion-3', input: 'koishi, masterpiece, best quality', parameters: 
{ seed: 5140, n_samples: 1, ucPreset: 2, qualityToggle: false, scale: 11, steps: 28, height: 768, width: 512, sampler: 'k_euler', image: undefined, negative_prompt: 'nsfw, lowres, {bad}, error, fewer, extra, missing, worst quality, jpeg artifacts, 
bad quality, watermark, unfinished, displeasing, chromatic aberration, signature, extra digits, artistic error, username, scan, [abstract]', sm_dyn: undefined, sm: undefined, noise_schedule: 'native' } }
2024-02-04 19:47:17 [I] novelai request { model: 'nai-diffusion-3', input: 'koishi, masterpiece, best quality', parameters: 
{ seed: 5140, n_samples: 1, ucPreset: 2, qualityToggle: false, scale: 11, steps: 28, height: 768, width: 512, sampler: 'k_euler', image: undefined, negative_prompt: undefined, sm_dyn: undefined, sm: undefined, noise_schedule: 'native' } }
```

**Comment by MirrorCY** [comment]
https://github.com/koishijs/novelai-bot/commit/3b09b3324d0b3c47867c80a3a12c689e56a93a2f#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R317-R320

**Comment by MirrorCY** [comment]
fixed in #221
